modules/ROOT/pages/jquery.adoc

```diff
@@ -0,0 +1,125 @@
+= TinyMCE jQuery integration quick start guide
+
+:title_nav: jQuery
+:description: Documentation for the official TinyMCE jQuery integration.
+:keywords: integration integrate jquery javascript
+:productSource: cloud
+
+The https://github.com/tinymce/tinymce-jquery[Official {productname} jQuery component] integrates {productname} into jQuery projects.
+
+
+NOTE: {companyname} are pleased to provide integrations/code guidance to help you build great products with {productname}. If you have queries about this integration, post your question on Stack Overflow using the link:{communitysupporturl}[`{prodnamecode}` tag].
```

Comment:
I'm pretty sure the company name is not a plural :)

```suggestion
NOTE: {companyname} is pleased to provide integrations/code guidance to help you build great products with {productname}. If you have queries about this integration, post your question on Stack Overflow using the link:{communitysupporturl}[`{prodnamecode}` tag].
```

Reply:
Good catch.